multibranch/src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java

```diff
@@ -71,7 +71,15 @@ public WorkflowMultiBranchProject(ItemGroup parent, String name) {
     @Extension public static class DescriptorImpl extends MultiBranchProjectDescriptor {
 
         @Override public String getDisplayName() {
-            return "Multibranch Pipeline";
+            return Messages.WorkflowMultiBranchProject_DisplayName();
+        }
+
+        public String getDescription() {
+            return Messages.WorkflowMultiBranchProject_Description();
+        }
+
+        public String getIconFilePathPattern() {
+            return "plugin/workflow-multibranch/images/:size/pipelinemultibranchproject.png";
```

Comment:
`getCategoryId()` is being provided by `AbstractFolderDescriptor` due to `DescriptorImpl` extends of `AbstractFolderDescriptor`.